MIP41/mip41.md

```diff
@@ -139,9 +139,15 @@ When the Facilitator of a Core Unit becomes unavailable, an existing permanent F
 
 The Interim Facilitator controls the budget to continue doing regular payouts to core Contributors and pay for other critical expenses and infrastructure while a permanent Core Unit Facilitator is sought.
 
+If a Core Unit has no active facilitators as a result of involuntary offboarding and has no active budget, no Interim Facilitator will be appointed unless there are winding down activities (such as returning assets to MakerDAO) that require one. An Interim Facilitator shall not be considered an active facilitator for purposes of MIP39c3.
+
 #### The Fallback Decision Process
 
-Interim Facilitators are designated based on the Fallback Decision Process, where a simple majority of all currently active Facilitators have to reach an agreement. The Fallback Decision Process can instantly designate an existing Facilitator (who must agree to the decision) and assign them as an Interim Facilitator to a Core Unit that is without a Facilitator, or that has a Facilitator who has gone missing.
+Interim Facilitators are designated based on the Succession Plan publicly documented by a Core Unit at onboarding. An Interim Facilitator appointed based upon a succession plan must be a current or former member of that Core Unit.
```

Comment:
(Is it a Fallback Decision Process if it's been properly covered by the succession plan? I guess that depends on the definition of "interim Facilitator", maybe

Reply:
ACK
I think this belongs under the "interim facilitators" header instead. 

> [](https://mips.makerdao.com/mips/details/MIP41#the-fallback-decision-process) The Interim Facilitator controls the budget to continue doing regular payouts to core Contributors and pay for other critical expenses and infrastructure while a permanent Core Unit Facilitator is sought. 

... Interim Facilitators are designated based on the...etc.